src/lib/api/EventAPI.ts

```diff
@@ -145,3 +146,28 @@ export const uploadEventImage = async (
     },
   });
 };
+
+export const awardBonusPoints = async (
+  token: string,
+  user: string,
```

Comment:
Is this a username or user UUID?

```suggestion
  username: string,
```

```suggestion
  user: UUID,
```

(or `userId: UUID`)